optimum/exporters/openvino/convert.py

```diff
@@ -81,6 +87,10 @@ def _save_model(model, path: str, ov_config: Optional["OVConfig"] = None):
 
         compress_to_fp16 = ov_config.dtype == "fp16"
 
+    model.set_rt_info(_transformers_version, ["optimum", "transformers_version"])
+    model.set_rt_info(_torch_version, ["optimum", "pytorch_version"])
+    model.set_rt_info(_optimum_intel_version, ["optimum", "optimum_intel_version"])
+    model.set_rt_info(_optimum_version, ["optimum", "optimum_version"])
```

Comment:
do we need additionally add info about nncf here?
and as some other libs may be used as model source for conversion (diffusers for sd, timm for image classification and sentence_transformers), I think it can be useful to provide this libs info too (if they are available or based on source lib for conversion)

Reply:
+1 for including NNCF version

Reply:
I added NNCF, diffusers, timm, sentence-transformers and onnx (for models converted from ONNX). I prefer not to add them just when the dependency is available, it's a bit strange to me to have timm_version in a BERT model for example. For timm and diffusers I now get the library name from the model path. Seems to work well, but let me know if there is a better way.